src/ui/liferea_shell.c

```diff
@@ -1259,6 +1259,7 @@ liferea_shell_create (GtkApplication *app, const gchar *overrideWindowState, gin
 	gtk_widget_show_all (shell->statusbar_feedsinfo_evbox);
 	gtk_box_pack_start (GTK_BOX (shell->statusbar), shell->statusbar_feedsinfo_evbox, FALSE, FALSE, 5);
 	g_signal_connect (G_OBJECT (shell->statusbar_feedsinfo_evbox), "button_release_event", G_CALLBACK (on_next_unread_item_activate), NULL);
+	g_signal_connect (G_OBJECT (shell->statusbar), "style-updated", G_CALLBACK(liferea_shell_rebuild_css), NULL);
```

Comment:
Found the issue. I'm a user of the "floating statusbar plugin", which hides the status bar when it is unused to save vertical space. And registering the signal to a widget that is not always visible seems to be the problem.

If I register to the shell->window the signal works properly for me. 

I'm guessing you are not using the plugin :-)

Still let's register the signal to "shell->window" to eliminate this corner case.

Actually this raised the next corner case: the main window being invisible (when hidden to tray icon). Then the signal is also lost...

Reply:
I remember thinking of this and that i went looking for a "View -> Hide Statusbar" option in the menu.

Reply:
I think i chose something other than the main window for a reason, also i think i've found the issue with the slighter darker badges...

Reply:
I hope you don't mind me moving the signal for now to the mainwindow. If you come across the reason for not having it there please raise a bug ticket so we can follow up on it. For now I want to have this great PR merged for 1.13.3 :-)